src/App.jsx

```diff
@@ -1,9 +1,32 @@
-import { HomePage } from './pages';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { ProtectedRoute } from './components';
+import { Header } from './components/organisms/Header/Header';
+import { ProfilePage, EditProfilePage, ChatPage, FeedScreenPage, ErrorPage, HomePage } from './pages';
 
 export const App = () => {
+    const someID="lolo" //temporary 
```

Comment:
constants should be declared outside of the component, if they don't have any dependencies with elements inside of the component

Also I know that it's temporary but please change name of the variable to something meaningfull for reference

Like: `const profileId = 'ab-2-4'`

Reply:
Done